fastly/resource_fastly_service_v1.go

```diff
@@ -989,7 +989,19 @@ func resourceServiceV1() *schema.Resource {
 							Type:        schema.TypeString,
 							Optional:    true,
 							Default:     "",
-							Description: "A secure certificate to authenticate the server with.",
+							Description: "A secure certificate to authenticate the server with. Must be in PEM format.",
+						},
+						"tls_client_cert": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Default:     "",
+							Description: "The client certificate used to make authenticated requests. Must be in PEM format.",
+						},
+						"tls_client_key": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Default:     "",
+							Description: "The client private key used to make authenticated requests. Must be in PEM format.",
```

Comment:
I think we should add the [sensitive schema method](https://www.terraform.io/docs/extend/schemas/schema-methods.html) to `tls_client_key`.

For consistency, we should also use [schema.EnvDefaultFunc](https://godoc.org/github.com/hashicorp/terraform/helper/schema#EnvDefaultFunc) on `tls_ca_cert`, `tls_client_cert`, and `tls_client_key` so that you can also use environtment variables directly.

Reply:
Good point, I've added these in efe58de856dc463f1228bfd80b0a588281c03d3a and 800ace912eeefbab048d6af8a316a691917fad60 (forgot to remove a `default:` param in the first commit